From 0c6636d8268f325d7cfaef3852951f47b018174b Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Tue, 4 Nov 2025 11:44:03 -0800
Subject: [PATCH] docs: w1: fix w1-netlink invalid URL
Git-commit: 0c6636d8268f325d7cfaef3852951f47b018174b
Patch-mainline: v6.19-rc1
References: git-fixes

The URL in w1-netlink.rst for an example userspace application
has changed. The former URL is no longer valid. Update it to the
github URL.

Fixes: e4e056aa3518 ("w1: documentation update")
Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-id: <20251104194403.945611-1-rdunlap@infradead.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 Documentation/w1/w1-netlink.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/w1/w1-netlink.rst b/Documentation/w1/w1-netlink.rst
index be4f7b82dcb4..ff281713e626 100644
--- a/Documentation/w1/w1-netlink.rst
+++ b/Documentation/w1/w1-netlink.rst
@@ -196,7 +196,7 @@ Additional documentation, source code examples
 ==============================================
 
 1. Documentation/driver-api/connector.rst
-2. http://www.ioremap.net/archive/w1
+2. https://github.com/bioothod/w1
 
    This archive includes userspace application w1d.c which uses
    read/write/search commands for all master/slave devices found on the bus.
-- 
2.52.0

